### ISSUE TITLE
Beef IDE 0.42.3 Right click doesn't open menus

### ISSUE THREAD

**Issue by Mario1159** [beefytech/Beef#228]
(Might be related to #187 , but i have the problem without a second monitor.)
Right click doesn't open menus, for example right clicking the Workspace doesn't display the menu that have buttons:
Open Containing Folder, Add New Project..., Add Existing Project, etc
The only part where display a menu is in the background where are located the tabs (like in the following image):
![imagen](https://user-images.githubusercontent.com/24532325/81099427-e8522880-8ed8-11ea-91e6-2b0dd64eab98.png)
OS: Windows 7 Professional

**Comment by bfiete** [comment]
Fixed for you?

**Comment by bfiete** [comment]
In 0.42.4, that is

**Comment by Mario1159** [comment]
Still having the issue

**Comment by bfiete** [comment]
Does the autocomplete popup work? Just not the right-click menus?

**Comment by Mario1159** [comment]
The autocomplete works, but right-clicking inside the text editor also doesn't work, it just trigger the yellow border for the time i hold the right click.
![imagen](https://user-images.githubusercontent.com/24532325/82711532-a6a0dc00-9c53-11ea-9993-4483323f2bf1.png)

**Comment by bfiete** [comment]
Also those drop shadows aren't even rending correctly. What kind of machine is this? This isn't in a VM or anything interesting?

**Comment by Mario1159** [comment]
No, I just have Windows installed, just note that my laptop its a bit old, but I'm having the problem just in the Beef IDE, I tried lauching with my integrated Intel Core HD Graphics 3000 and with the Nvidia Geforce 610M but it both GPUs i have the same results.
